.github/workflows/build-publish.yaml

```diff
@@ -36,11 +36,20 @@ jobs:
       - name: Setup dependencies
         run: go mod download
       - name: Run tests
+        run: go test -race ./...
+
+  test-internal:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Dispatch workflow
         env:
-          GH_USER: nestobot
           GH_TOKEN: ${{ secrets.TESTING_TOKEN }}
-          CATALOG_URL: https://github.com/nestoca/catalog
-        run: go test ./...
+        run: |
+          gh workflow run --repo github.com/nestoca/catalog -f ref=${{ github.head_ref }} joy-generator-test
+          sleep 5s
+          workflowId=$(gh run ls --repo github.com/nestoca/catalog --workflow joy-generator-test --json databaseId -q '.[0].databaseId')
+          gh run watch --exit-status --repo github.com/nestoca/catalog $workflowId
```

Comment:
Is this going to print the logs of the job? That would be no bueno, we will want to only see a pass/fail and nothing else, to avoid anything leaking accidentally.

Reply:
It only prints the jobs of the workflow and their status: Pending, in-progress, completed, error.

So if a workflow has build, publish, release jobs you would see those jobs successively complete or fail!